djangoplicity/archives/contrib/admin/defaults.py

```diff
@@ -181,7 +181,7 @@ def _check_embargo(model, obj, request):
 
 def _get_ordered(obj, klass):
     fields = [(name, o) for name, o in list(obj.__dict__.items()) if isinstance(o, klass)]
-    fields.sort(lambda tuple1, tuple2: cmp(tuple1[1].creation_counter, tuple2[1].creation_counter))
+    fields.sort(key=lambda field: field[1].creation_counter)
```

Comment:
I don't believe this is a correct upgrade, please refer to the upgrades we have already performed:

https://github.com/djangoplicity/djangoplicity/wiki/Python-3-Upgraded-Lines

Reply:
I see you've updated this line before but related to a different issue. And you also provided this solution to another issue:
```python
countries_list.sort(key=lambda x: x[0])
```
So, what could be the solution to this issue?

Reply:
I don't know, perhaps a unit test for it would be required, or debugging it

Reply:
All right, I'll try to create a unit test for this or debug it in a local instance to see if it works

Reply:
@EdisonArango  I started an ESAHubble development server to test this solution and it's now showing any error but it's not showing any content either. So I don't know if this fixed the issue.

![image](https://user-images.githubusercontent.com/9884128/95619329-131a4d00-0a34-11eb-996b-cf22d80cd1b6.png)

Reply:
Hi @EdisonArango and @javierriveros.

@danialeresbar and me were testing this solution and we realize that newsletter's display action is always showing an empty 'Translation list' even for the current production site (spacetelescope.org). This is maybe because esahubble doesn't use translations. However, announcements also have a displays action that fails because of exactly the same `sort` line and after testing this solution, we get the next result:

![image](https://user-images.githubusercontent.com/36459619/102796007-9680e480-437b-11eb-84c9-1ef4b48437e9.png)

Based on this result we think this is correct.

Reply:
https://user-images.githubusercontent.com/36459449/102798103-a2ba7100-437e-11eb-90cf-8d0fc41fe21f.mp4